htcanalyze/main.py

```diff
@@ -393,45 +368,32 @@ def manage_params(args: list) -> dict:
         new_filter_list.extend(li)
     cmd_dict["filter_keywords"] = new_filter_list
 
-    # parse the mode correctly
-    if commands_parsed.analyze_mode and commands_parsed.summarize_mode:
-        mode = "analyzed-summary"
-    elif commands_parsed.analyze_mode:
-        mode = "analyze"
-    elif commands_parsed.summarize_mode:
-        mode = "summarize"
-    elif commands_parsed.mode is not None:
-        if commands_parsed.mode in ALLOWED_MODES.keys():
-            mode = ALLOWED_MODES[commands_parsed.mode]
-        else:
-            mode = commands_parsed.mode
-    else:
-        mode = None  # will result in default mode
-
-    cmd_dict["mode"] = mode
     # error handling
     try:
-        if cmd_dict["filter_extended"] and not cmd_dict["filter_keywords"]:
-            raise_value_error("--extend not allowed without --filter")
-        if cmd_dict["show_list"] and (
-                mode == "analyzed-summary" or mode == "summarize"):
+        if cmd_dict["show_list"] and not cmd_dict["one_by_one"]:
             raise_value_error("--show only allowed with analyze mode")
     except ValueError as err:
         rprint(f"[red]htcanalyze: error: {err}[/red]")
         sys.exit(2)
 
     # delete unnecessary information
-    del cmd_dict["summarize_mode"]
-    del cmd_dict["analyze_mode"]
     del cmd_dict["version"]
     del cmd_dict["no_config"]
 
     if cmd_dict['ext_log'] is None:
         cmd_dict['ext_log'] = ""
+    elif not cmd_dict['ext_log'].startswith('.'):
+        cmd_dict['ext_log'] = '.' + cmd_dict['ext_log']
```

Comment:
`cmd_dict['ext_log] = f".{cmd_dict['ext_log']"`. Note that this will cause problems on non-linux platforms. I'm not sure if that's relevant though, but maybe worth mentioning somewhere.